server/svix-server/src/v1/endpoints/application.rs

```diff
@@ -39,6 +39,17 @@ use serde::{Deserialize, Serialize};
 use svix_server_derive::ModelOut;
 use validator::{Validate, ValidationError};
 
+#[derive(Deserialize, JsonSchema)]
+struct ApplicationIdOrUidParam {
```

Comment:
Nit-pick: but no need to be too verbose here. The "IdOrUid" is implied because it's a path param and we always accept both.

`ApplicationPath` would be enough.
Then for endpoint:
`ApplicationEndpointPath`
Then for attempt:
`ApplicationMsgAttemptPath`

I think this is consistent and easy.